printer_mods/conqueror433/Voron_Parts_Logo/README.md

```diff
@@ -0,0 +1,23 @@
+Voron Parts Logo for your VORON Printer
+
+STL File - You can print this logo on your 3D printer. I have included base (1mm thick) in this stl file which you can print in base color (at 0.2mm layer height) 
+           & change filament after 5 layers to your accent color
+
+Image Files - Image files are included in high quality JPG & PNG format if anyone wants to print them on paper with traditional ink printer.
+
+DXF File - DXF file is included if anyone wants to cut this on laser cutter out of Vinyl wrap. (Like I did, Shown in Images below)
+
+
+![P_20210304_231717](https://user-images.githubusercontent.com/74809512/110663581-952dbe00-81ec-11eb-881b-88fae0d535d8.jpg)
```

Comment:
All these images that are included in your README - can you add them directly to this pull request?  You can put them in a `printer_mods/conqueror433/Voron_Parts_Logo/images/` directory for example.  Then update the image URLs in the README to relative links to those images, so your README images would be something like

    ![P_20210304_231717](images/some_file_name.jpg)

Reply:
Done